shell/shared/grain.js

```diff
@@ -411,7 +411,11 @@ Router.map(function () {
         result.appOrigin = document.location.protocol + "//" + makeWildcardHost(session.hostId);
         setCurrentSessionId(session._id, result.appOrigin, grainId);
         result.sessionId = session._id;
-        result.path = encodeURIComponent("/" + (self.params.path || ""));
+        var currentPath = Iron.Location.get().path;
```

Comment:
I don't think I want to use window.location as a string since I need to parse out the pathname and hash anyways. It seems much better to use window.location.pathname and window.location.hash directly